app/istqb-ctel-tm-otm-complete-guide/istqb-ctel-tm-otm-complete-guide.css

```
@@ -0,0 +1,969 @@
+            @keyframes pulse {
+                0%,
+                100% {
+                    opacity: 0.6;
+                    transform: translate(-50%, -50%) scale(1);
+                }
+                50% {
+                    opacity: 1;
+                    transform: translate(-50%, -50%) scale(1.1);
+                }
+            }
+            @keyframes fadeInDown {
```

Stylelint (17.12.0)

[error] Expected keyframe name "fadeInDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)